src/api/app/components/add_review_dropdown_component.rb

```diff
@@ -0,0 +1,32 @@
+class AddReviewDropdownComponent < ApplicationComponent
+  def initialize(bs_request:, user:)
+    super
+
+    @bs_request = bs_request
+    @user = user
+  end
+
+  def render?
+    can_add_reviews?
+  end
+
+  def reviews
+    @bs_request.reviews.where(state: 'new')
+  end
+
+  def can_add_reviews?
```

Comment:
This is a `Policy`

Reply:
Things meanwhile moved, but your comment is still valid in other places. We will document/comment the things that should go into a policy and tackle this in an follow up.